CONTRIBUTING.MD

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+1. Create a Python virtual environment
+
+2. Install the identity package in editable mode:
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+
+3. Run the tests:
+
+    ```bash
+    python -m pytest
+    ```
+
+4. Make the docs:
+
+    ```bash
+    python -m mkdocs build
```

Comment:
Does it require some extra dependency in requirements.txt?

Reply:
Yepyep youre right, added
pip install -r docs/requirements.txt